examples/src/keyrings/aws_kms_discovery_keyring_example.py

```diff
@@ -0,0 +1,201 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example sets up the AWS KMS Discovery Keyring
+
+AWS KMS discovery keyring is an AWS KMS keyring that doesn't specify any wrapping keys.
+
+The AWS Encryption SDK provides a standard AWS KMS discovery keyring and a discovery keyring
+for AWS KMS multi-Region keys. For information about using multi-Region keys with the
+AWS Encryption SDK, see
+https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/configure.html#config-mrks
+
+Because it doesn't specify any wrapping keys, a discovery keyring can't encrypt data.
+If you use a discovery keyring to encrypt data, alone or in a multi-keyring, the encrypt
+operation fails. The exception is the AWS Encryption SDK for C, where the encrypt operation
+ignores a standard discovery keyring, but fails if you specify a multi-Region discovery
+keyring, alone or in a multi-keyring.
```

Comment:
```suggestion
operation fails. 
```

This isn't super relevant for Python, let's leave that in the docs